src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,80 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        File file = new File(toFileName);
+        file.delete();
+        String readFromFileString = readFromFile(fromFileName);
+        String[] makeReportArray = makeReport(readFromFileString);
+        writeToFile(makeReportArray, toFileName);
+    }
+
+    private String readFromFile(String fromFileName) {
+        StringBuilder stringBuilder = new StringBuilder();
+        try {
```

Comment:
[Use try with resources when you work with classes that implement AutoCloseable interface](https://mate-academy.github.io/jv-program-common-mistakes/java-core/builder-file/work-with-file#try-with-resources)